### PR TITLE
About Me steps 1-4

### PR DESCRIPTION
**Pull request by joverde**
First 4 Steps of the About me project here!

### CODE REVIEW

**Comment by aeftenie** [comment]
Can you please reach out to either Emily or Lauren and ask them if they can help review this PR?

**Review by laurencvossler — APPROVED** [review]
Looks good to me!

File: src/main/webapp/js/user-page-loader.js [hunk 1]
```diff
@@ -39,6 +39,9 @@ function showMessageFormIfLoggedIn() {
       })
       .then((loginStatus) => {
         if (loginStatus.isLoggedIn) {
+            if (loginStatus.username == parameterUsername){
```
Comment (aeftenie) [inline src/main/webapp/js/user-page-loader.js]: Alignment looks a bit off. Shouldn't the `if` on `line 42` align with the `const` on `line 45`?

File: src/main/webapp/js/user-page-loader.js [hunk 2]
```diff
@@ -39,6 +39,9 @@ function showMessageFormIfLoggedIn() {
...
+                document.getElementById('about-me-form').classList.remove('hidden');
```
Comment (aeftenie) [inline src/main/webapp/js/user-page-loader.js]: Can you break this line similar to the "message-form" element below to improve readability?

```
const aboutMeForm = document.getElementById('about-me-form');
aboutMeForm.classList.remove('hidden');
```

File: src/main/webapp/js/user-page-loader.js (hunk 1 above)
Comment (aeftenie) [inline src/main/webapp/js/user-page-loader.js]: I'm not sure why the about me page is under an `if (loginStatus.username == parameterUsername)` statement. Shouldn't a user be able to see the about me page of others too?

**Review by aeftenie — CHANGES_REQUESTED** [review]

File: src/main/webapp/js/user-page-loader.js (hunk 1 above)
Comment (emilytran99) [reply src/main/webapp/js/user-page-loader.js]: Does this mean that you just have to be logged in to view any user's about me page?

**Review by emilytran99 — COMMENTED** [review]
This looks good to me! Don't have any changes

**Review by emilytran99 — APPROVED** [review]
Oops didn't realize comment and approve were different things -- sorry if this shows up twice

File: src/main/webapp/js/user-page-loader.js (hunk 1 above)
Comment (aeftenie) [reply src/main/webapp/js/user-page-loader.js]: @emilytran99 I think that's guarded by the `if (loginStatus.isLoggedIn)` statement. Josh can correct me though.

Comment (joverde) [reply src/main/webapp/js/user-page-loader.js]: I believe what this does is only allow the user to have the ability to make the about me post request if viewing their own account, so they can only update their own about me sections.

File: src/main/webapp/js/user-page-loader.js [hunk 3]
```diff
@@ -39,6 +39,10 @@ function showMessageFormIfLoggedIn() {
       })
       .then((loginStatus) => {
         if (loginStatus.isLoggedIn) {
+          if (loginStatus.username == parameterUsername){
+            const AboutMeClassList = document.getElementById('about-me-form').classList;
```
Comment (aeftenie) [inline src/main/webapp/js/user-page-loader.js]: We should use the same casing (camelCase) for all const variables.

Comment (aeftenie) [inline src/main/webapp/js/user-page-loader.js]: This does not maintain the format of the `messageForm` const below where only the "form" element is saved to a constant. Saving the classList object on the form only seems to make the code more verbose and repetitive.

**Review by aeftenie — CHANGES_REQUESTED** [review]

File: src/main/webapp/js/user-page-loader.js (hunk 1 above)
Comment (aeftenie) [reply src/main/webapp/js/user-page-loader.js]: Got it! Thanks for the clarification!

**Review by aeftenie — APPROVED** [review]

**Pull request merged**